### DISCUSSION TITLE
API Documentation

### DISCUSSION THREAD

**Discussion by thetronjohnson** [hoppscotch/hoppscotch#4182] [Ideas]
Hi 👋 

We're in the process of planning an API documentation feature for Hoppscotch and would appreciate your input to help us understand the current challenges.

1. How do you currently document your APIs?
2. Do you document APIs for internal use, or do you publish them for other developers to use?
3. What do you like about your current API documentation tool?
4. What challenges do you face while documenting your APIs?

Thank you for your feedback!

**Comment by baek-sang** [comment]
https://github.com/hoppscotch/hoppscotch/issues/3964

https://github.com/hoppscotch/hoppscotch/issues/2018

https://github.com/hoppscotch/hoppscotch/pull/4075

**Comment by baek-sang** [comment]
https://github.com/hoppscotch/hoppscotch/pull/4075

Why are there no reviews for this PR?
